src/test/java/uk/gov/cshr/controller/signup/SignupControllerTest.java

```diff
@@ -210,7 +211,7 @@ public void shouldSendToLoginIfSignupCodeNotValid() throws Exception {
                 get("/signup/" + code)
                         .with(CsrfRequestPostProcessor.csrf()))
                 .andExpect(status().is3xxRedirection())
-                .andExpect(redirectedUrl("/login"));
+                .andExpect(redirectedUrl("/signup/request"));
```

Comment:
This expect is not in synch with the unit test method name. Method name suggests that expectation is to redirect to "Login" rather than signup. Please either correct the method name or correct the expectation.

Reply:
Because of the change in the workflow the user is being navigated to signup page now so the the reason for above change. Also I have changed the method name to reflect this change.